src/api/app/views/webui/project/keys_and_certificates.html.haml

```diff
@@ -0,0 +1,44 @@
+- @pagetitle = "Key and certificate for #{@project}"
+
+.key-and-certificates-card
+  = render partial: 'tabs', locals: { project: @project }
+  .card-body
+    %h3.card-title= @pagetitle
+    .row
+      .gpg_key_column
+        .card
+          .card-body
+            %h5.card-title GPG Key
+            %dl.row
+              %dt.size-term Size
+              %dd.size-definition= @project.key_info.keysize
+
+              %dt.algorithm-term Algorithm
+              %dd.algorithm-definition= @project.key_info.algorithm
+
+              %dt.id-term ID
+              %dd.id-definition= @project.key_info.keyid
+
+              - if @project.key_info.expires
+                %dt.expires-term Expires
+                %dd.expires-definition= DateTime.strptime(@project.key_info.expires, '%s').strftime('%F')
+
+              - if @project.key_info.origin.present?
+                %dt.origin-term Origin
+                %dd.origin-definition= @project.key_info.origin
+            %p.download-link
+              = link_to(project_public_key_path(project_name: @project.name)) do
+                %i
+                GPG Key
+      .ssl_cert_column
+        .card
+          .card-body
+            %h5.card-title SSL Certificate
+            %dl
+              %dt Fingerprint
+              %dd= @project.key_info.fingerprint
```

Comment:
This is the fingerprint of the GPG key, it has nothing to do with the SSL certificate.

Reply:
True, fixing

Reply:
Fixed